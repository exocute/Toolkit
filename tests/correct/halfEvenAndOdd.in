Graph HalfEvenAndOdd
	Import Vector[Int]
	Export Boolean
	Activity A SplitVector
	Import Vector[Int]
	Export Vector[Int]
	Activity B TestEven
	Import Vector[Int]
	Export Boolean
	Activity C TestOdd
	Import Vector[Int]
	Export Boolean
	Activity D HalfEvenAndOdd
	Import Boolean
	Import Boolean
	Export Boolean
	
	Connection [A;B,C] [B;D] [C;D]