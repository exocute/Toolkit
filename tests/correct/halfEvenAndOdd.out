Name: HalfEvenAndOdd
ImportName: Vector[Int]
ExportName: Boolean
Id: D, Name: HalfEvenAndOdd, Import: Boolean,Boolean, Export: Boolean
Id: A, Name: SplitVector, Import: Vector[Int], Export: Vector[Int]
Id: C, Name: TestOdd, Import: Vector[Int], Export: Boolean
Id: B, Name: TestEven, Import: Vector[Int], Export: Boolean
A:B,C
C:D
B:D